src/mofdscribe/featurizers/chemistry/loading.py

```diff
@@ -0,0 +1,227 @@
+                Defaults to False.
+
+        Raises:
+            ValueError: If the `RASPA_DIR` environment variable is not set.
+        """
+        self.raspa_dir = raspa_dir if raspa_dir else os.environ.get("RASPA_DIR")
+        if self.raspa_dir is None:
+            try:
+                self.raspa_dir = detect_raspa_dir()
+            except ValueError:
+                raise ValueError(
+                    "Please set the RASPA_DIR environment variable or provide the path for the class initialization."
+                )
+        self.cycles = cycles
+        self.cutoff = cutoff
+        self.mof_ff = mof_ff
+        self.mol_ff = mol_ff
+        self.mol_name = mol_name
+        self.tail_corrections = tail_corrections
+        self.mixing_rule = mixing_rule
+        self.shifted = shifted
+        self.separate_interactions = separate_interactions
+        self.temperature = temperature
+        self.pressure = pressure
+        self.run_eqeq = run_eqeq
+        self.return_std = return_std
+        super().__init__(primitive=primitive)
+
+    def _featurize(self, s: Union[Structure, IStructure]) -> np.array:
+        ff_molecules = {self.mol_name: self.mol_ff}
+
+        parameters = {
+            "ff_framework": self.mof_ff,
+            "ff_molecules": ff_molecules,
+            "shifted": self.shifted,
+            "tail_corrections": self.tail_corrections,
+            "mixing_rule": self.mixing_rule,
+            "separate_interactions": self.separate_interactions,
+        }
+        replicas = resize_unit_cell(s, self.cutoff)
+        ucells = f"{replicas[0]} {replicas[1]} {replicas[2]}"
+        simulation_script = LOADING_INPUT_TEMPLATE.format(
+            cycles=self.cycles,
+            unit_cells=ucells,
+            cutoff=self.cutoff,
+            print_every=self.cycles // 10,
+            molname=self.mol_name,
+            temp=self.temperature,
+            press=self.pressure,
+        )
+
+        res, deviations = run_raspa(
+            s,
+            self.raspa_dir,
+            simulation_script,
+            parameters,
+            parse_loading,
+            self.run_eqeq,
+        )
+
+        if self.return_std:
+            res.extend(deviations)
+        return np.array(res)
+
+    def feature_labels(self) -> List[str]:
+        feat = [
+            f"loading_{self.mol_name}_{self.temperature}_{self.pressure}_mol/kg",
+            f"loading_excess_{self.mol_name}_{self.temperature}_{self.pressure}_mol/kg",
+        ]
+        if self.return_std:
+            feat.extend(
+                [
+                    f"loading_std_{self.mol_name}_{self.temperature}_{self.pressure}_mol/kg",
+                    f"loading_excess_std_{self.mol_name}_{self.temperature}_{self.pressure}_mol/kg",
+                ]
+            )
+        return feat
+
+    def implementors(self) -> List[str]:
+        return ["Fergus Mcilwaine", "Kevin Maik Jablonka", "David Dubbeldam and RASPA authors"]
```

Comment:
```suggestion
        return ["Fergus Mcilwaine"]
```


i think the implementors mostly refer to the implementor of the featurizer, which is you in this case